core/templates/css/oppia.css

```diff
@@ -3228,6 +3228,9 @@ md-card.preview-conversation-skin-supplemental-card {
   list-style-type: none;
   text-align: center;
 }
+.oppia-navbar-nav.oppia-navbar-profile > li:hover {
+  background-color: #00645c
+}
```

Comment:
I see that it has been brought down from the top-navigation bar component. Was it not working before?